src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -313,7 +313,6 @@ public void Dispose()
         }
 
         [Fact]
-        [ActiveIssue(18128, platforms: TestPlatforms.AnyUnix)] // NotSupportedException thrown instead of InvalidOperationException
```

Comment:
It was correct because the other assertions in the test were failing, but also partially wrong as it didn't mention anything about flushasync, I think.